pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -2877,6 +2877,20 @@ type CephFilesystemSubVolumeGroupSpec struct {
 	// list of Ceph Filesystem volumes with `ceph fs volume ls`. To learn more about Ceph Filesystem
 	// abstractions see https://docs.ceph.com/en/latest/cephfs/fs-volumes/#fs-volumes-and-subvolumes
 	FilesystemName string `json:"filesystemName"`
+	// Pinning configuration of CephFilesystemSubVolumeGroup,
+	// reference https://docs.ceph.com/en/quincy/cephfs/fs-volumes/#pinning-subvolumes-and-subvolume-groups
+	// +optional
+	Pinning CephFilesystemSubVolumeGroupSpecPinning `json:"pinning,omitempty"`
+}
+
+// CephFilesystemSubVolumeGroupSpecPinning represents the pinning configurtion of SubVolumeGroup
+type CephFilesystemSubVolumeGroupSpecPinning struct {
+	// Pinning type of CephFilesystemSubVolumeGroup
+	// +optional
+	PinType string `json:"pinType,omitempty"`
+	// Pinning setting of CephFilesystemSubVolumeGroup
+	// +optional
+	PinSetting int `json:"pinSetting,omitempty"`
```

Comment:
Let's set 
pinType 
- default to distributed.
- make it enum with values random, export and distributed

and 
pinSetting
- default to 1 
@batrick Do we have any min and max value for <pin_setting> ?

Reply:
> @batrick Do we have any min and max value for <pin_setting> ?

It depends on which pin type you're setting. export can be <-1 (disabled), 0-256>. distributed <0, 1>, and random [0.0, 1.0].

Reply:
@batrick so it fine to add 
default values to all subvolume group:  pinType: `distributed` pinSetting:`1`

Reply:
Yes, that is the default behavior we want.

Reply:
> Yes, that is the default behavior we want.

@batrick Can multiple pin type be set at once(e.g. export to 200 & distributed to 1) or only one can be chosen at a time(if distributed is 1 , export is set to -1 and random to 0 automatically)? 
Just confirming since current design allows only one settings in Customer resource.